pinc/send_mail.inc

```diff
@@ -0,0 +1,153 @@
+<?php
+include_once($relPath.'site_vars.php');
+include_once($relPath.'misc.inc'); // html_safe()
+include_once($relPath.'Project.inc');
+include_once($relPath.'User.inc');
+
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\SMTP;
+use PHPMailer\PHPMailer\Exception;
+
+function send_mail($to, $subject, $message, $from = null, $reply_to = null)
+// If this is for real, send the message.
+// If we're testing, just report what would have been sent.
+// Optional $from and $reply_to arguments may either take the form
+// "user@example.com" or "Example User <user@example.com>"
+{
+    global $testing, $phpmailer_smtp_config, $auto_email_addr, $site_name, $site_url;
+
+    $from = $from ?? $auto_email_addr;
+    $reply_to = $reply_to ?? $auto_email_addr;
+
+    // Append a standard footer to all emails sent out by the system.
+    if (!endswith($message, "\n")) {
+        $message .= "\n";
+    }
+
+    $message .= implode("\n", [
```

Comment:
Perhaps not for this PR, but I'd love for us to consider are more refined footer for HTML emails since we have complete control over that here.

For instance, perhaps:
```html
<hr>
<a href="$site_url">$site_name</a><br>
<small>This is an automated message.</small>
```

Reply:
Did a small amount of prettifying in latest commit